thetis/stability_functions.py

```diff
@@ -99,41 +112,51 @@ def __init__(self, lim_alpha_shear=True, lim_alpha_buoy=True,
                 self.description += ['ac'+str(self.alpha_buoy_crit)]
         self.description = ' '.join(self.description)
 
-        # Umlauf and Buchard (2005) eq A.10
-        self.a1 = 2.0/3.0 - 0.5*self.cc2
-        self.a2 = 1.0 - 0.5*self.cc3
-        self.a3 = 1.0 - 0.5*self.cc4
-        self.a4 = 0.5*self.cc5
-        self.a5 = 0.5 - 0.5*self.cc6
-
-        self.ab1 = 1.0 - self.cb2
-        self.ab2 = 1.0 - self.cb3
-        self.ab3 = 2.0*(1.0 - self.cb4)
-        self.ab4 = 2.0*(1.0 - self.cb5)
-        self.ab5 = 2.0*self.cbb*(1.0 - self.cb5)
-
-        # Umlauf and Buchard (2005) eq A.12
-        self.nn = 0.5*self.cc1
-        self.nb = self.cb1
-
-        # Umlauf and Buchard (2005) eq A.9
-        self.d0 = 36.0*self.nn**3*self.nb**2
-        self.d1 = 84.0*self.a5*self.ab3*self.nn**2*self.nb + 36.0*self.ab5*self.nn**3*self.nb
-        self.d2 = 9.0*(self.ab2**2 - self.ab1**2)*self.nn**3 - 12.0*(self.a2**2 - 3.0*self.a3**2)*self.nn*self.nb**2
-        self.d3 = 12.0*self.a5*self.ab3*(self.a2*self.ab1 - 3.0*self.a3*self.ab2)*self.nn +\
-            12.0*self.a5*self.ab3*(self.a3**2 - self.a2**2)*self.nb +\
-            12.0*self.ab5*(3.0*self.a3**2 - self.a2**2)*self.nn*self.nb
-        self.d4 = 48.0*self.a5**2*self.ab3**2*self.nn + 36.0*self.a5*self.ab3*self.ab5*self.nn**2
-        self.d5 = 3.0*(self.a2**2 - 3.0*self.a3**2)*(self.ab1**2 - self.ab2**2)*self.nn
-        self.n0 = 36.0*self.a1*self.nn**2*self.nb**2
-        self.n1 = -12.0*self.a5*self.ab3*(self.ab1 + self.ab2)*self.nn**2 +\
-            8.0*self.a5*self.ab3*(6.0*self.a1 - self.a2 - 3.0*self.a3)*self.nn*self.nb +\
-            36.0*self.a1*self.ab5*self.nn**2*self.nb
-        self.n2 = 9.0*self.a1*(self.ab2**2 - self.ab1**2)*self.nn**2
-        self.nb0 = 12.0*self.ab3*self.nn**3*self.nb  # NOTE ab3 is not squared!
-        self.nb1 = 12.0*self.a5*self.ab3**2*self.nn**2
-        self.nb2 = 9.0*self.a1*self.ab3*(self.ab1 - self.ab2)*self.nn**2 +\
-            (6.0*self.a1*(self.a2 - 3.0*self.a3) - 4.0*(self.a2**2 - 3.0*self.a3**2))*self.ab3*self.nn*self.nb
+        # denominator
+        self.d0 = 1.0
+        self.d1 = 1.0
+        self.d2 = 1.0
+        self.d3 = 1.0
+        self.d4 = 1.0
+        self.d5 = 1.0
+
+        # c_mu
+        self.n0 = 0.0
+        self.n1 = 0.0
+        self.n2 = 0.0
+
+        # c_mu_p
+        self.nb0 = 0.0
+        self.nb1 = 0.0
+        self.nb2 = 0.0
+
+    def compute_alpha_shear_steady(self, ri_st, analytical=True):
```

Comment:
Does this need a docstring rather than a description in comments?

Reply:
yes, fixed.